Adds new question for the javascript interview

pixxstudios opened the pull request:
**Updates the coding-question.md file to add Javascript coding question.**




Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant.

- [x ] New Question


# Checklist:

- [ x] My content follows the style guidelines of this project
- [ x] I have performed a self-review of my own content

Pull requests should be thought of as a conversation. There will be some back and forth when trying to get code merged into this or any other project. With all but the simplest changes you can and should expect that the maintainers of the project will request changes to your code. Please be aware of that and check in after you open your PR in order to get your code merged in cleanly. 

Thanks!

darcyclarke commented:
@pixxstudios appreciate the contribution. Just wondering how we could potentially change this question a bit to help better articulate the concept you're asking of the interview (ie. prototypal inheritance). Maybe we could change this example/snippet to not include an error/bug out of the box. Maybe something like this.

---

*Question: Can you update the example code below to add a new `drive()` function to the `Car` object? This function should simply `console.log` some value when it's executed.*

```javascript
function Car() {}
// your code here...
const honda = new Car();
honda.drive(); // this should console.log some value 
```

darcyclarke reviewed (changes requested):
@pixxstudios appreciate the contribution. Just wondering how we could potentially change this question a bit to help better articulate the concept you're asking of the interview (ie. prototypal inheritance). Maybe we could change this example/snippet to not include an error/bug out of the box. Maybe something like this.

---

*Question: Can you update the example code below to add a new `drive()` function to the `Car` object? This function should simply `console.log` some value when it's executed.*

```javascript
function Car() {}
// your code here...
const honda = new Car();
honda.drive(); // this should console.log some value 
```

pixxstudios commented:
Hi Darcy,

Thanks for the feedback. I have updated the description and the question format based on your suggestion and updated the pull request.


Thanks.
